templates/home/reports/pre-a11y-1-fr.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Pre-accessibility assessment #1",
+	"language": "fr",
+	"description": "Assessing the main content of the GC Home Page template",
+	"tag": "home",
+	"parentdir": "home",
+	"parentPage": "Accueil - Canada.ca",
+	"parentPageURL": "home-fr.html",
+	"altLangPage": "a11y-1-en.html",
```

Comment:
This page don't exist

```suggestion
	"altLangPage": "pre-a11y-1-en.html",
```